test/test_train_mp_imagenet_amp.py

```diff
@@ -219,23 +219,29 @@ def train_imagenet():
       summary_writer=writer)
   loss_fn = nn.CrossEntropyLoss()
   if FLAGS.amp:
-    scaler = GradScaler(use_zero_grad=FLAGS.use_zero_grad)
+    if device_hw == 'TPU':
+      scaler = None
+    elif device_hw == 'GPU':
+      scaler = GradScaler(use_zero_grad=FLAGS.use_zero_grad)
```

Comment:
Now we have a full AMP story, I think we should add a doc under https://github.com/pytorch/xla/tree/master/docs to explain how to use it on TPU and GPU. We should also mentioned tricks like `GradScaler` here. WDYT?

I had a https://github.com/pytorch/xla/blob/master/docs/gpu.md#amp-automatic-mixed-precision here but I think it is better to have a standalone doc we can refer to during release.

Reply:
I agree, will add some documentation.